### ISSUE TITLE
Vertical measured size does not include Horizontal Scrollbar

### ISSUE THREAD

**Issue by garyhertel** [AvaloniaUI/AvaloniaEdit#144]
If I use an AvaloniaEdit control inside a Grid and set the RowDefinition Height to Auto, the AvaloniaEdit control doesn't seem to factor in the scrollbar at the bottom.

Note how the scrollbar is on top of the last line:

![image](https://user-images.githubusercontent.com/32446542/126877664-4a4f0348-8f5c-48b6-bbbc-04b7f6c34ef4.png)

I modified the AvaloniaEdit.Demo to reproduce this:
```
    <Grid RowDefinitions="Auto, *" ColumnDefinitions="*">
        <AvalonEdit:TextEditor Name="Editor"
                              VerticalAlignment="Top" HorizontalAlignment="Stretch"
                               Background="Blue"
                               Foreground="White"
                               HorizontalScrollBarVisibility="Visible"
                               VerticalScrollBarVisibility="Visible"/>
        <Panel VerticalAlignment="Stretch" HorizontalAlignment="Stretch" Grid.Row="2" Background="Green"/>
    </Grid>
```

And commented out the buttons in the MainWindow so it would compile with the changes

I had previously worked around this by overriding the TextEditor MeasureOverride() to add an extra 12 pixels to the height, but that seems to cause more problems than it solves after updating from 0.10.0 to 0.10.4, with the height gradually growing when resizing.

**Comment by danipen** [comment]
@garyhertel I think this is not an AvaloniaEdit bug. 

The AvaloniaEdit demo application uses the Avalonia Fluent Theme. The scrollbars in FluentTheme overlays the controls. This is the default behavior.

![image](https://user-images.githubusercontent.com/501613/127509294-d7187113-4dff-471f-87ab-970833cfabdd.png)

**Comment by garyhertel** [comment]
Thanks! Yeah, I think you're right and I was looking at the wrong issue.

Looking at this again, I think my actual problem is with the vertical size suddenly jumping when selecting all the text. Fortunately, it does look like this was fixed sometime after the AvaloniaEdit 0.10.4 release. I don't suppose we could get a shiny new version sometime? :)
